lib/main.dart

```diff
@@ -0,0 +1,193 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const HearthstoneApp());
+
+class HearthstoneApp extends StatelessWidget {
+  const HearthstoneApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: "Hearthstone App",
+      theme: ThemeData(
+        fontFamily: "MedievalSharp",
+        scaffoldBackgroundColor: Colors.orange[100]
+      ),
+      home: const HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  int _likeCounter = 0;
+
+  void addLike(){
+    setState(() {
+      _likeCounter++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Center(
+          child: ListView(
+              children: [
+                Image.asset(
+                    'assets/images/ysera_card.png',
+                  height: 500,
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    const Icon(
+                      Icons.favorite,
+                      color: Colors.pink,
+                      size: 30,
+                    ),
+                    const SizedBox(
+                      width: 10,
```

Comment:
width can be constant

Reply:
Done

I wrapped the Text Widget into a Padding but added EdgeInsets.only(left: 10) for spacing between Icon and Counter value. No need for margin top or bottom.